docs/source/server-process.md

```diff
@@ -153,6 +153,18 @@ One of:
 
 (server-process:callable-arguments)=
 
+### `websockify`
```

Comment:
Let's call this something else - perhaps `raw_tcp_proxy_only` (or something else, am not tied to that)? Otherwise there'll be confusion about what it means to 'use jupyter-server-proxy with websockify'.

Reply:
Oh, also mention that this would then bypass the code we have that allows folks to rewrite responses.

Reply:
Happy to rename it whatever people feel makes sense.

It doesn't exactly bypass the response rewriting, but the rewriting only is used for normal http responses anyway, not websockets, so because all other requests return 405 it never gets used. I guess we could call rewrite on the 405 response? But yes, I'll clarify this, too.

Reply:
I don't want to get into too much of a naming rabbit hole, but I'll suggest `raw_tcp_proxy` here (without the `only`). Any objections to that, @dylex?

Reply:
Sounds fine to me. It is specifically using websockets but I guess that should be clear.

Reply:
@dylex Let's do `raw_socket_proxy`!